### ISSUE TITLE
Focal loss in AVA detection

### ISSUE THREAD

**Issue by hukkai** [open-mmlab/mmaction2#1477]
Hello, not sure if it is a bug or more likely I misunderstood the codes. Thanks ahead for your clarification!

As I read the focal loss paper and other implementations, the alpha hyper-parameter is set between 0 and 1. But I saw the value here is 3: https://github.com/open-mmlab/mmaction2/blob/6f79f54a34fd53c18adf4577fc59ab934f5c7d07/configs/detection/ava/slowfast_temporal_max_focal_alpha3_gamma1_kinetics_pretrained_r50_8x8x1_cosine_10e_ava22_rgb.py#L43

From another issue I knew it is obtained by parameter search, so I go to the codes for focal loss. In this line https://github.com/open-mmlab/mmaction2/blob/e9f141520bbafaf3cde3f3d35279ee4430bc4d86/mmaction/models/heads/bbox_head.py#L250 self.focal_alpha is a scalar multiplier for both positive and negative classes. So it only changes the length of the gradient but not the direction? I think the alpha_t should be alpha for positive and 1-alpha for negative samples, i.e.,
https://github.com/kuangliu/pytorch-retinanet/blob/2d7c663350f330a34771a8fa6a4f37a2baa52a1d/loss.py#L35
 Please correct me if I am wrong. Thanks!

**Comment by kennymckormick** [comment]
Yes, in our code focal_alpha is just a scalar multiplier, our implementation of focal_loss is a little bit different due to the setting: AVA is a multi-label classification problem and a binarycrossentropyloss is adopted, which is unlike single-label classification with CELoss.
